### ISSUE TITLE
[Visual feature request] Show the 3x3 visual grid just like the hammer from TiC

### ISSUE THREAD

**Issue by Hund** [ColinGBos/Hammerz#18]
The hammers from Tinkers Construct shows a visual 3x3 grid when you point the hammer to a wall, making it easy to see what you're mining.

It would be nice if you could consider adding this to your mod, with a config option to disable/enable it ofcourse. :)

**Comment by ColinGBos** [comment]
Yeah I'll try something like this. Will be a while before I get around to it though.

**Comment by Hund** [comment]
Thanks!
